arcade/sound.py

```diff
@@ -58,7 +58,7 @@ def play(
         player: media.Player = media.Player()
         player.volume = volume
         player.position = (pan, 0.0, math.sqrt(1 - math.pow(pan, 2)))  # used to mimic panning with 3D audio
-        player.pitch = pitch
+        player.pitch = speed # note that the underlying attribute is pitch but "speed" is used because it better describes the behavior see #1198
```

Comment:
It's probably best to break this up as a multiple comment lines above setting the pitch on the player. The linters won't like this as-is.